nilearn/datasets/struct.py

```diff
@@ -122,7 +122,7 @@ def fetch_icbm152_2009(data_dir=None, url=None, resume=True, verbose=1):
     return Bunch(**params)
 
 
-def load_mni152_template(resolution=1):
+def load_mni152_template(resolution=2):
     """Load the skullstripped 1mm-resolution version of the MNI152 T1 template,
```

Comment:
Thanks! We also need to update the docstrings to reflect this change.

Reply:
I just made a small update in the docstring but note that we are actually taking the 1mm-resolution template.